cloudformation/backend/app-infrastructure.yml

```diff
@@ -190,6 +190,8 @@ Resources:
               Value: '{{resolve:ssm:UPGRADE_TYPEORM_SYNCHRONIZE}}'
             - Name: USE_NEW_RELIC
               Value: '{{resolve:ssm:UPGRADE_USE_NEW_RELIC}}'  
+            - Name: IS_ECS
+              Value: '{{resolve:ssm:UPGRADE_IS_ECS}}'
```

Comment:
Can just set the value to `true` since it ill always be true for this deployment, no need to use a ssm parameter.